spartan/scripts/test_kind.sh

```diff
@@ -57,6 +57,8 @@ fi
 # If fresh_install is true, delete the namespace
 if [ "$fresh_install" = "true" ]; then
   echo "Deleting existing namespace due to FRESH_INSTALL=true"
+  # Run helm uninstall first to ensure post-delete hooks are run
```

Comment:
Could you ELI5 this to me? I'm not sure I fully understand the interaction between helm and kubectl here.

Reply:
my understanding is that namespace delete forcefully shuts down & deletes everything while helm uninstall follows helm charts & that includes running hooks (in this case we need a post-delete hook to run).

Reply:
my understanding is that namespace delete forcefully shuts down & deletes everything while helm uninstall follows helm charts & that includes running hooks (in this case we need a post-delete hook to run).